docs/source/api_docs/index.rst

```diff
@@ -9,8 +9,7 @@ This page provides a list of all publicly accessible modules, methods and classe
     :maxdepth: 2
     :caption: API Documentation
 
-    gis
-    spatial_indexing
+    spatial
```

Comment:
Are trajectories not spatial?

Reply:
Good question, I assumed trajectories are spatial + temporal data. But I'm not strongly preferenced to keep that independent from `spatial`.

Reply:
That's a good point.